viam-cartographer/scripts/setup_cartographer_macos.sh

```diff
@@ -6,4 +6,3 @@ brew install abseil boost ceres-solver protobuf ninja cairo googletest lua@5.3
 brew link lua@5.3
 brew install openssl eigen gflags glog suite-sparse sphinx-doc pcl
 brew link protobuf
-brew link openssl --force
```

Comment:
🥳

Reply:
Spoke to @nicksanford offline and he asked me to test this by removing all brew packages (not just `openssl`) and making sure this line is still not needed. I did this and confirmed that the build works even without this. It did make me realize that we're requiring `pkg-config` and `cmake` without having the setup script install those, so I made a change to brew install those packages as well.